src/OpenCL.h

```diff
@@ -135,19 +142,27 @@ class OpenCL_Network {
     void batchnorm(int outputs, int channel_size, cl::Buffer& input,
                    cl::Buffer& output, cl::Buffer* residual,
                    weight_slice_t weights);
+
+    OpenCL * m_opencl;
```

Comment:
We know it's never null, and it's passed in the constructor. so we should be able to make this a reference instead of a raw pointer.